master/master.cfg

```diff
@@ -13,35 +13,37 @@
 import os
 import sys
 
+from datetime import timedelta
+
 from buildbot.schedulers.basic import SingleBranchScheduler
 from buildbot.schedulers.timed import Nightly
 from buildbot.schedulers.forcesched import (ForceScheduler, FixedParameter,
                                             StringParameter, UserNameParameter)
+from buildbot.plugins import reporters, util
 from buildbot.process import factory
-from buildbot.status import html, mail, words
 from buildbot.steps.shell import ShellCommand, Configure, Compile
 from buildbot.steps.transfer import DirectoryUpload
 from buildbot import locks
 from buildbot.steps.source.git import Git
+from buildbot.www.auth import RemoteUserAuth
 
 sys.path.append(os.path.dirname(__file__))   # noqa: E402
 
 from custom.steps import Test, Clean, Install, LockInstall, Uninstall
 
 from utils import changecommentlink
-from xmlrpc import XMLRPCServer
```

Comment:
You can also remove `xmlrpc.py` from the repo.

Reply:
Fixed